### PR TITLE
Release notes for 0.6 "I Accidentally The Smart Contract"

### PR DESCRIPTION
**Pull request by rustyrussell**
Written by Christian Decker wth feedback from Shannon Appelcline.
Special thanks to Fabrice Drouin of ACINQ fame for choosing the release
name!

### CODE REVIEW

File: doc/release-notes/release-0.6.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Release Announce for 0.6
+## a.k.a. "I Accidentally The Smart Contract"
+
+The long wait is over: we, the c-lightning team, are excited to announce the 0.6 release of
+[c-lightning][clightning], an important milestone for the project. This complete rewrite of the previous implementation is the first fully specification-compilation release of c-lightning. It migrates away from the protocol used while designing the specification and toward a new architecture that is modular and extensible, to better adapt to your needs and your infrastructure.
```
Comment (cdecker) [inline doc/release-notes/release-0.6.md]: specification-compilation -> specification-compliant

Must have snuck in while copy-editing

Comment (wythe) [reply doc/release-notes/release-0.6.md]: The first line might be stronger with fewer commas.  How about:

The wait is over.  The c-lighting team is pleased to announce release 0.6, an important milestone for the Lightning Network.

Comment (rustyrussell) [reply doc/release-notes/release-0.6.md]: Agreed with both.

**Comment by cdecker** [comment]
ACK 036fdd9

**Pull request merged**